builder/main.js

```diff
@@ -58,11 +84,53 @@ Code:
 ${snippet.toString()}
 \`\`\`
 `);
+
   document.querySelector(
     "calcite-button"
   ).href = `${repoUrl}/issues/new?title=${title}&body=${body}`;
 }
 
+document.getElementById("import-snippet").addEventListener("change", (event) => {
+  const file = event.target.files[0];
+
+  if (file) {
+    const reader = new FileReader();
+
+    reader.onload = function(e) {
+      try {
+        const fileContent = e.target.result;
+        const snippetObject = JSON.parse(fileContent);
+
+        const snippetDisplay = document.getElementById("snippets");
+        snippetDisplay.innerText = JSON.stringify(snippetObject, null, 2);
```

Comment:
This is good, I'm just missing the form to be filled too:
![2024-10-16_16-08-29](https://github.com/user-attachments/assets/4ae5a7ff-88da-4a6d-b710-4ea72baf0811)